packages/vaex-contrib/vaex/contrib/io/gbq.py

```diff
@@ -0,0 +1,194 @@
+'''
+A module for I/O between Vaex and Google BigQuery.
+
+Requires:
+ - google.cloud.bigquery
+ - google.cloud.bigquery_storage
+'''
+
+import tempfile
+
+import pyarrow as pa
+import pyarrow.parquet as pq
+
+import vaex.utils
+
+google = vaex.utils.optional_import("google", modules=[
+    "google.cloud.bigquery",
+    "google.cloud.bigquery_storage",
+])
+
+
+def query_google_big_query(query, client_project=None, credentials=None):
```

Comment:
I think it's already namespace enough, we don't need to repeat gbq in the function names, and to make it more standard, like vaex.from_*
```suggestion
def from_query(query, client_project=None, credentials=None):
```